src/components/movie-details/movie-details.test.tsx

```diff
@@ -0,0 +1,50 @@
+import '@testing-library/jest-dom'
+import { render, screen } from '@testing-library/react'
+import { describe, expect, it } from 'vitest'
+import { MovieDetails } from './movie-details.tsx'
```

Comment:
Seems like this should work

```suggestion
import { render, screen } from '@testing-library/react'
import { MovieDetails } from './movie-details.tsx'
```